webhook.rst

```diff
@@ -189,13 +445,26 @@ For SMS webhooks, react to the
         }
     }
 
-Creating a Custom Webhook
--------------------------
 
-.. tip::
+Providing Webhooks
+------------------
 
-    Starting in `MakerBundle`_ ``v1.58.0``, you can run ``php bin/console make:webhook``
-    to generate the request parser and consumer files needed to create your own
-    Webhook.
+Symfony Webhook and Symfony Remote Event, when combined with Symfony Messenger, are also useful for APIs responsible for dispatching webhooks.
 
-.. _`MakerBundle`: https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html
+For instance, you can utilize the specific :class:`SendWebhookMessage` and :class:`SendWebhookHandler` provided to dispatch the message either synchronously or asynchronously using the Symfony Messenger component.
+
+The SendWebhookMessage takes a :class:`Subscriber` as its first argument, which includes the destination URL and the mandatory secret.
+If the secret is missing, an exception will be thrown.
+
+As a second argument, it expects a :class:`RemoteEvent` containing the webhook name, the ID, and the payload, which is the substantial information you wish to communicate.
+
+The :class:`SendWebhookHandler` configures the headers, the body of the request, and finally sign the headers before making an HTTP request to the specified URL using Symfony's HttpClient component::
+
+    $subscriber = new Subscriber($urlCallback, $secret);
+
+    $event = new Event(‘name.event, ‘1’, […]);
+
+    $this->bus->dispatch(new SendWebhookMessage($subscriber, $event));
+
```

Comment:
add for a customised approach using the `Symfony\Component\Webhook\Server\TransportInterface`